tools/external_ic.F90

```diff
@@ -1817,6 +1821,11 @@ subroutine get_ecmwf_ic( Atm, fv_domain )
       jsd = Atm%bd%jsd
       jed = Atm%bd%jed
 
+      call open_ncfile( trim(inputdir)//'/'//trim(fn_gfs_ctl), ncid )
+      call get_ncdim1( ncid, 'levsp', levsp )
+      call close_ncfile( ncid )
+      levp_gfs = levsp-1
+
```

Comment:
Will need to go through and fix these at some point to use fms2_io.

Reply:
Since this is the only change that would require me to re-test, I chose not to address this immediately.  Would it be okay for me to put in an issue here on Github and once this is released I can work to update this?

Reply:
This was only a note to point out something to be addressed at a later date.